data-prepper-plugins/opensearch/src/test/java/org/opensearch/dataprepper/plugins/sink/opensearch/BulkRetryStrategyTests.java

```diff
@@ -68,7 +68,7 @@ public void testCanRetry() {
         AccumulatingBulkRequest accumulatingBulkRequest = mock(AccumulatingBulkRequest.class);
```

Comment:
Which one of these tests are validating we are honoring the configured retry config?

Reply:
Added new tests to test retry config.